source/speech/speech.py

```diff
@@ -1861,6 +1860,127 @@ def getPropertiesSpeech(  # noqa: C901
 	return textList
 
 
+def speakKeyboardShortcuts(keyboardShortcutsStr: Optional[str]) -> None:
+	speak(getKeyboardShortcutsSpeech(keyboardShortcutsStr))
+
+
+def getKeyboardShortcutsSpeech(keyboardShortcutsStr: Optional[str]) -> SpeechSequence:
+	"""Gets the speech sequence for a shortcuts string containing one or more shortcuts.
+	@param keyboardShortcutsStr: the shortcuts string.
+	"""
+	
+	SHORTCUT_KEY_LIST_SEPARATOR = '  '
+	seq = []
+	if not keyboardShortcutsStr:
+		return seq
+	try:
+		for shortcutKeyStr in keyboardShortcutsStr.split(SHORTCUT_KEY_LIST_SEPARATOR):
+			seq.extend(_getKeyboardShortcutSpeech(shortcutKeyStr))
+			seq.append(SHORTCUT_KEY_LIST_SEPARATOR)
+		seq.pop()  # Remove last SHORTCUT_KEY_LIST_SEPARATOR in the sequence
+	except Exception:
+		log.warning(
+			f'Error parsing keyboard shortcut "{keyboardShortcutsStr}", reporting the string as a fallback.',
+			exc_info=True,
+		)
+		return [keyboardShortcutsStr]
+
+	# Merge consecutive strings in the list when possible
+	seqOut = []
+	for item in seq:
+		if len(seqOut) > 0 and isinstance(seqOut[-1], str) and isinstance(item, str):
+			seqOut[-1] = seqOut[-1] + item
+		else:
+			seqOut.append(item)
+
+	return seqOut
+
+
+def _getKeyboardShortcutSpeech(keyboardShortcut: str) -> SpeechSequence:

```

Comment:
it would be great if simple unit tests were added to these helper functions, just to document what they do.